templates/partials/modals/import.html

```
@@ -1,5 +1,5 @@
 <!-- Import Modal -->
-<div id="import-modal" class="modal" role="dialog" aria-label="Import Settings" aria-modal="true">
+<div id="import-modal" class="modal" role="dialog" aria-label="Import Settings" aria-modal="true" aria-hidden="true">
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)